src/components/patient/ConfirmationModal.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Button, Modal, Typography } from '@mui/material'
+import { DataGrid } from '@mui/x-data-grid'
+import { UpdatedFields } from './PatientDetails'
+
+const style = {
+  position: 'absolute' as 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 600,
+  bgcolor: 'background.paper',
+  boxShadow: 24
+}
+
+const columns = [
+  {
+    field: 'field',
+    headerName: 'Field',
+    flex: 1
+  },
+  {
+    field: 'original',
+    headerName: 'Original',
+    flex: 1
+  },
+  {
+    field: 'new',
+    headerName: 'New',
+    flex: 1
+  }
+]
+
+const ConfirmationModal: React.FC<{
+  isVisible: boolean
+  handleClose: () => void
+  onConfirm: () => void
+  updatedFields: UpdatedFields[]
+}> = ({ isVisible, handleClose, updatedFields, onConfirm }) => {
+  return (
+    <Modal
+      open={isVisible}
+      onClose={handleClose}
+      aria-labelledby="parent-modal-title"
+      aria-describedby="parent-modal-description"
+    >
+      <Box
+        id="modal-description"
+        sx={{
+          ...style,
```

Comment:
@IlyesBenAmara isn't there any prop that allows the center the modal content instead of using css ?